tests/unit/modules/test_linux_lvm.py

```diff
@@ -316,7 +316,7 @@ def test_lvcreate(self):
         )
 
         mock = MagicMock(return_value="A\nB")
-        with patch.dict(linux_lvm.__salt__, {"cmd.run": mock}):
```

Comment:
This should return a dict now. Also it needs 2 tests: one for out and one for err.

Reply:
Thanks. I have updated and added tests.

Reply:
Thanks. I have updated and added tests.